packages/config/config/devices/0x045a/Z-CM-V01.json

```diff
@@ -0,0 +1,218 @@
+{
+	"manufacturer": "ZVIDAR",
+	"manufacturerId": "0x045a",
+	"label": "Z-CM-V01",
+	"description": "Smart Curtain Motor",
+	"devices": [
+		{
+			"productType": "0x0904",
+			"productId": "0x0507"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Set To Start Holding Hands",
+			"description": "This parameter can be used to set the motor open hand start function",
```

Comment:
This is a weird label. What does this actually do? I don't really understand.

Reply:
I honestly have no idea what this means either. I just rolled with it because there was another curtain motor that had the same label:
https://github.com/zwave-js/node-zwave-js/blob/918904ae7a80be2460fbb9bd8a5285e304f016cf/packages/config/config/devices/0x030a/dt82tv_f-1_2_14.json#L39

I have tried setting this value thru the UI, but as far as I can tell, it doesn't change any behavior.